admin/services/redis-cache.service.js

```diff
@@ -87,4 +87,29 @@ redisCacheService.drop = async (caches = []) => {
   return true
 }
 
+/**
+ * @param {[{key:string, value:object}]} batch a dictionary of items to add to redis
+ * @param {number} ttl time to expiry in seconds, optional.
+ * @returns {Promise<void>}
+ */
+redisCacheService.addBatch = async (batch, ttl = 0) => {
```

Comment:
I think it's more accurate if ttl defaults to undefined or null if not set.  We might actually want to set the ttl to 0.